runtime/jcl/exports.cmake

```diff
@@ -0,0 +1,577 @@
+		Java_com_ibm_tools_attach_target_IPC_createFileWithPermissionsImpl
+		Java_com_ibm_tools_attach_target_IPC_destroySemaphore
+		Java_com_ibm_tools_attach_target_IPC_getProcessId
+		Java_com_ibm_tools_attach_target_IPC_getTempDirImpl
+		Java_com_ibm_tools_attach_target_IPC_getUid
+		Java_com_ibm_tools_attach_target_IPC_isUsingDefaultUid
+		Java_com_ibm_tools_attach_target_IPC_mkdirWithPermissionsImpl
+		Java_com_ibm_tools_attach_target_IPC_notifyVm
+		Java_com_ibm_tools_attach_target_IPC_openSemaphore
+		Java_com_ibm_tools_attach_target_IPC_processExistsImpl
+		Java_com_ibm_tools_attach_target_IPC_setupSemaphore
+		Java_com_ibm_tools_attach_target_IPC_tracepoint
+		Java_com_ibm_tools_attach_target_IPC_waitSemaphore
+	)
+endif()
+
+if(OMR_ENV_DATA32)
+	omr_add_exports(jclse
+		Java_com_ibm_jit_JITHelpers_getArrayShapeFromRomClass32
+		Java_com_ibm_jit_JITHelpers_getBackfillOffsetFromJ9Class32
+		Java_com_ibm_jit_JITHelpers_getClassDepthAndFlagsFromJ9Class32
+		Java_com_ibm_jit_JITHelpers_getClassFlagsFromJ9Class32
+		Java_com_ibm_jit_JITHelpers_getClassFromJ9Class32
+		Java_com_ibm_jit_JITHelpers_getDescriptionWordFromPtr32
+		Java_com_ibm_jit_JITHelpers_getInstanceDescriptionFromJ9Class32
+		Java_com_ibm_jit_JITHelpers_getJ9ClassFromClass32
+		Java_com_ibm_jit_JITHelpers_getModifiersFromRomClass32
+		Java_com_ibm_jit_JITHelpers_getRomClassFromJ9Class32
+		Java_com_ibm_jit_JITHelpers_getSuperClassesFromJ9Class32
+		Java_com_ibm_jit_JITHelpers_getTotalInstanceSizeFromJ9Class32
+		Java_com_ibm_oti_vm_ORBVMHelpers_getDescriptionWordFromPtr32
+		Java_com_ibm_oti_vm_ORBVMHelpers_getInstanceDescriptionFromJ9Class32
+		Java_com_ibm_oti_vm_ORBVMHelpers_getJ9ClassFromClass32
+		Java_com_ibm_oti_vm_ORBVMHelpers_getTotalInstanceSizeFromJ9Class32
+	)
+else()
+	omr_add_exports(jclse
+		Java_com_ibm_jit_JITHelpers_getArrayShapeFromRomClass64
+		Java_com_ibm_jit_JITHelpers_getBackfillOffsetFromJ9Class64
+		Java_com_ibm_jit_JITHelpers_getClassDepthAndFlagsFromJ9Class64
+		Java_com_ibm_jit_JITHelpers_getClassFlagsFromJ9Class64
+		Java_com_ibm_jit_JITHelpers_getClassFromJ9Class64
+		Java_com_ibm_jit_JITHelpers_getDescriptionWordFromPtr64
+		Java_com_ibm_jit_JITHelpers_getInstanceDescriptionFromJ9Class64
+		Java_com_ibm_jit_JITHelpers_getJ9ClassFromClass64
+		Java_com_ibm_jit_JITHelpers_getModifiersFromRomClass64
+		Java_com_ibm_jit_JITHelpers_getRomClassFromJ9Class64
+		Java_com_ibm_jit_JITHelpers_getSuperClassesFromJ9Class64
+		Java_com_ibm_jit_JITHelpers_getTotalInstanceSizeFromJ9Class64
+		Java_com_ibm_oti_vm_ORBVMHelpers_getDescriptionWordFromPtr64
+		Java_com_ibm_oti_vm_ORBVMHelpers_getInstanceDescriptionFromJ9Class64
+		Java_com_ibm_oti_vm_ORBVMHelpers_getJ9ClassFromClass64
+		Java_com_ibm_oti_vm_ORBVMHelpers_getTotalInstanceSizeFromJ9Class64
+	)
+endif()
+
+
+if(JAVA_SPEC_VERSION EQUAL 8)
+	omr_add_exports(jclse Java_sun_misc_URLClassPath_getLookupCacheURLs)
+endif()
+
+# java 9+
+if(NOT JAVA_SPEC_VERSION LESS 10)
```

Comment:
This doesn't match the comment: the exports below will be selected for java `10+`.
This should be:
```
if(NOT JAVA_SPEC_VERSION LESS 9)
```
The same goes below for the expression selecting java 11+.